praktikumsplaner-backend/src/main/resources/application-dev.yml

```diff
@@ -13,13 +13,7 @@ spring:
   h2.console.enabled: true
   jpa:
     database: H2
-    hibernate:
-        # always drop and create the db should be the best
-        # configuration for local (development) mode. this
-        # is also the default, that spring offers by convention.
-        # but here explicite:
-        ddl-auto: create-drop
```

Comment:
i would change this to `validate` and move it in the default profile (`application.yml`).